app/Http/Transformers/v3SignupTransformer.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Rogue\Http\Transformers;
+
+use Rogue\Models\Signup;
+use League\Fractal\TransformerAbstract;
+
+class v3SignupTransformer extends TransformerAbstract
+{
+    /**
+     * Transform resource data.
+     *
+     * @param \Rogue\Models\Signup $signup
+     * @return array
+     */
+    public function transform(Signup $signup)
+    {
+        // @TODO - This is temporary. We have migrated data that has stored quanity in the
+        // quanity_pending column on the signup. However, since then we updated the business
+        // logic to store everything in the quanity column and not use the quanity_pending
+        // column at all. We only want to return what is in the quanity_pending column
+        // if is the only place quanity is stored.
+        if (! is_null($signup->quantity_pending) && is_null($signup->quantity)) {
```

Comment:
One thing that'd be nice is to move this into the model (perhaps as an [accessor](https://laravel.com/docs/5.5/eloquent-mutators#defining-an-accessor) on the `quantity` field) so that we don't need this logic repeated in each of the transformers.

Reply:
great idea! I just updated this :)